[2024-12-19T08:47:14.218+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:47:14.235+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-10-01T00:00:00+00:00 [queued]>
[2024-12-19T08:47:14.248+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-10-01T00:00:00+00:00 [queued]>
[2024-12-19T08:47:14.250+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:47:14.274+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2021-10-01 00:00:00+00:00
[2024-12-19T08:47:14.288+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=725) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:47:14.290+0000] {standard_task_runner.py:72} INFO - Started process 732 to run task
[2024-12-19T08:47:14.289+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2021-10-01T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpe73vdq02']
[2024-12-19T08:47:14.292+0000] {standard_task_runner.py:105} INFO - Job 66: Subtask first_task
[2024-12-19T08:47:14.345+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2021-10-01T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:47:14.437+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-01T00:00:00+00:00'
[2024-12-19T08:47:14.439+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:47:14.457+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:47:14.459+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:47:14.474+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:47:14.476+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:47:14.477+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:47:14.501+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:47:14.502+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2021-10-01T00:00:00+00:00, execution_date=20211001T000000, start_date=20241219T084714, end_date=20241219T084714
[2024-12-19T08:47:14.549+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:47:14.563+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:47:14.566+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
